libfreerdp/codec/dsp.c

```diff
@@ -1307,6 +1325,22 @@ BOOL freerdp_dsp_context_reset(FREERDP_DSP_CONTEXT* context, const AUDIO_FORMAT*
 		return FALSE;
 
 	context->format = *targetFormat;
+
+	if (context->format.wFormatTag == WAVE_FORMAT_DVI_ADPCM)
+	{
+		size_t min_frame_data =
+		    context->format.wBitsPerSample * context->format.nChannels * FramesPerPacket;
```

Comment:
like the codeql error tells. simply add `*1ULL` or cast all to `size_t` to fix.

Reply:
Done